doc/api/next_api_changes/2019-01-19-AL.rst

```diff
@@ -0,0 +1,10 @@
+Deprecations
+````````````
+
+Support for passing colors as UPPERCASE strings is deprecated; color names will
+become case-sensitive after the deprecation period has passed.
```

Comment:
Just for clarity! You should add that all buildings color names are lowercase.

Reply:
done